### ISSUE TITLE
Cannot delete user

### ISSUE THREAD

**Issue by sgofferj** [TryGhost/Ghost#12790]
Welcome to Ghost's GitHub repo! 👋🎉

We use GitHub only for bug reports 🐛

Anything else should be posted to https://forum.ghost.org 👫

🚨For support, help & questions use https://forum.ghost.org/c/help
💡For feature requests & ideas you can post and vote on https://forum.ghost.org/c/Ideas

If your issue is with Ghost CLI, please report it on the CLI repo ➡️ https://github.com/TryGhost/Ghost-CLI/issues/new.

### Issue Summary
I signed up on my own self-hosted ghost site with a different email address to test how the subscription works. When I tried to delete the user after testing. it showed the error

```
Internal server error, cannot delete member. delete from `members` where `id` = '6053b1344f2ec70001088ca1' - SQLITE_ERROR: foreign key mismatch - "members_stripe_customers_subscriptions" referencing "members_stripe_customers"
```

I don't have any stripe connection because I don't plan any paid subscriptions.

### To Reproduce

1. Sign up a user for a free plan (no paid plan availabe)
2. Try to delete the user

Well, I consider it a bug because when I click on "delete user" I expect the user to be deleted and not see an internal server error.
### Technical details:

* Ghost Version: 4.01 - official docker image
* Node Version: Unknown
* Browser/OS: Chrome & Firefox on Ubunti 20.04
* Database: sqlite

**Comment by daniellockyer** [comment]
Hey @sgofferj! 👋🏻 This was fixed with https://github.com/TryGhost/Ghost/pull/12784 and released in 4.1.0. You'll need to update to that when the Docker image is updated 🙂

**Comment by sgofferj** [comment]
Thanks!
